bud.config.js

```diff
@@ -1,17 +1,17 @@
 /**
  * Compiler configuration
  *
- * @see {@link https://roots.io/docs/sage sage documentation}
- * @see {@link https://bud.js.org/guides/configure bud.js configuration guide}
+ * @see {@link https://roots.io/sage/ sage documentation}
```

Comment:
I think `https://roots.io/sage/docs/` is a more direct link to the docs here.

Reply:
I agree. I'll close this one and create new one.

Reply:
the link is now`https://roots.io/sage/docs/` in line 4